continuationsPlugin/src/main/scala/continuations/constants.scala

```diff
@@ -1,8 +1,15 @@
 package continuations
 
+import dotty.tools.dotc.ast.tpd.*
+import dotty.tools.dotc.core.Contexts.Context
+import dotty.tools.dotc.core.Names.termName
+import dotty.tools.dotc.core.Symbols.requiredClass
+
 private[continuations] val continuationPackageName = "continuations"
+private[continuations] val suspendContinuationName = "suspendContinuation"
 private[continuations] val suspendFullName = s"$continuationPackageName.Suspend"
 private[continuations] val continuationFullName = s"$continuationPackageName.Continuation"
-private[continuations] val suspendContinuationFullName =
-  s"$continuationFullName.suspendContinuation"
 private[continuations] val resumeFullName = s"$continuationPackageName.Continuation.resume"
+
+private[continuations] def suspendContinuationMethod(using Context): Select =
```

Comment:
not a constant per se but not sure where else it would make sense to have it

Reply:
Let's create a `Trees` trait and put it under that, and mix it in where we need it. We could probably extract a lot of common Tree's there, eventually.

Reply:
done in [f01e32f](https://github.com/47deg/TBD/pull/58/commits/f01e32f3053eabccc7eedc642106aced40394ebb)